src/components/CalculatorLine.test.js

```diff
@@ -0,0 +1,62 @@
+import { fireEvent, render, screen } from '@testing-library/react';
+import CalculatorLine from './CalculatorLine';
+
+describe('Input Render', () =>{
```

Comment:
You can use as many describe as you want to organize your code.
I would suggest adding the `Input Render` and Input validation` into another describe block with the component name. This way when you look at the test reports you won't need to look for the file name